sdk/nodejs/batch/v1beta1/CronJob.ts

```diff
@@ -100,6 +100,13 @@ import { getVersion } from "../../version";
               opts.version = getVersion();
           }
 
-          super(CronJob.__pulumiType, name, props, opts);
+          const _opts = pulumi.mergeOptions(opts, {
+              aliases: [
+                  { parent: opts.parent, type: "kubernetes:batch/v1beta1:CronJob", name: name },
+                  { parent: opts.parent, type: "kubernetes:batch/v2alpha1:CronJob", name: name },
```

Comment:
Interesting that there is a `v2` here - does that not indicate some fundamental incompatibility?  Does Kubernetes treat these as sharing identity across `v1`/`v2`?

Reply:
I don't see any explicit guidance in the [upstream docs](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning). However, it probably doesn't make sense to auto-alias to `alpha` apiVersions since it's possible that they have backwards-incompatible changes.

Reply:
Actually, we've already been auto-aliasing a number of `alpha` apiVersions, so let's keep it for now, and adjust in the future if we discover that it causes problems.